ynr/apps/parties/importer.py

```diff
@@ -0,0 +1,324 @@
+from urllib.parse import urlencode, urljoin
+import re
+from datetime import datetime
+from tempfile import NamedTemporaryFile
+import mimetypes
+import magic
+import dateutil.parser
+
+import requests
+
+from .constants import (
+    EC_API_BASE,
+    EC_EMBLEM_BASE,
+    DEFAULT_EMBLEMS,
+    JOINT_DESCRIPTION_REGEX,
+    CORRECTED_PARTY_NAMES_IN_DESC,
+    CORRECTED_DESCRIPTION_DATES,
+)
+from .models import Party, PartyDescription, PartyEmblem
+
+
+def extract_number_from_id(party_id):
+    m = re.search("\d+", party_id)
+    if m:
+        return int(m.group(0), 10)
+
+
+def make_joint_party_id(id1, id2):
+    numbers = sorted(map(extract_number_from_id, [id1, id2]))
+    return "joint-party:{}-{}".format(*numbers)
+
+
+class ECPartyImporter:
+    def __init__(self):
+        self.collector = []
+        self.base_url = EC_API_BASE
+        self.per_page = 50
+        self.params = {
+            "rows": self.per_page,
+            "et": ["pp", "ppm"],
+            "register": ["gb", "ni", "none"],
+            "regStatus": ["registered", "deregistered", "lapsed"],
+            "period": [
+                "127",
```

Comment:
Joking aside, could we have a comment here explaining what this list of arbitrary numbers actually means.